src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -196,7 +196,9 @@ public static Week parseWeek(String week) throws ParseException {
      * @throws ParseException if the given {@code indices} is invalid.
      */
     public static List<Index> parseIndices(String indices) throws ParseException {
-        requireNonNull(indices);
+        if (indices.isEmpty()) {
+            return new ArrayList<>();
```

Comment:
I cant think of any scenarios, but do you think this would cause any bugs when we do include pre/ but dont type anything after?

Reply:
Yeah spent some time on this bug yesterday actually. Now, the behaviour is that if you enter "pre/" and nothing after, it tells you that you are using the command incorrectly.